hw/formal/tools/dvsim/common_conn_cfg.hjson

```diff
@@ -0,0 +1,55 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  flow:             fpv
+  fpv_root:         "{proj_root}/hw/formal"
```

Comment:
I'm a bit unsure about the name here: FPV stands for "formal property verification", which seems reasonable. Calling this "formal" seems to imply we don't ever expect any other formal flow, which doesn't seem right to me. "fconn", maybe?

Reply:
Yeah that is a good point. We do not expect any other formal flow to follow the exact flow as fconn and fpv. I like the name `fconn`! Thanks for the input. I modified the flow name to `fconn`. For your comment, are you also concerning the folder name `{proj_root}hw/formal`?

Reply:
I would say use `flow: fpv` for the major ASIC flows (others are `sim`, `syn`, `lint`). Corresponding to each flow we have a `hw/{flow}` directory - so the directory name should match the flow name. 

This is a sub-flow that runs under the 'fpv' blanket., so perhaps add `sub_flow: conn` to signify that. The other sub_flows under `fpv` are `secure_path`, `xprop`, `tb_analyzer` etc.

Reply:
You can reference the subflow-specific tcl scripts as `{fpv_root}/tools/{tool}/{subflow}.tcl`.

Reply:
Thanks Sri for your input as well. I think as Sri mentioned, keeping `formal` as generic flow sounds good. I was planning to change all generic `fpv` naming to `formal` in dvsim in another PR.
In terms of expecting all formal tools to have same flow as Rupert mentioned, I think we can still keep a generic formal flow that will:
1. Use fusesoc to create filelist
2. Run {subflow}.tcl script to trigger specific apps (eg: fpv, fconn, etc as Sri mentioned)
3. Parse results and errors. Currently we have `parse_fpv_errors.py` under jaspergold. I think we can reuse the script, as all jaspergold formal tools have same namings: proven, cex, unreachable etc.
Does that sound reasonable?

Reply:
Sounds reasonable. I sort of dislike equating "formal" with "Jasper", but I guess it's just reflecting the current state of the world :-)